packages/integration_test/lib/integration_test_driver.dart

```diff
@@ -16,11 +16,8 @@ import 'common.dart';
 
 /// Flutter Driver test output directory.
 ///
-/// Tests should write any output files to this directory. Defaults to the path
-/// set in the FLUTTER_TEST_OUTPUTS_DIR environment variable, or `build` if
-/// unset.
-String testOutputsDirectory =
-    Platform.environment['FLUTTER_TEST_OUTPUTS_DIR'] ?? 'build';
+/// Tests should write any output files to this directory. Defaults to `build`.
+String testOutputsDirectory = fs.systemTempDirectory.childDirectory('build').path;
```

Comment:
Here and above: Will this break anything? I see FLUTTER_TEST_OUTPUTS_DIR referenced in the flutter_tool a couple of times and google3 also seems to refer to this env var. I can't tell if anything will break if we stop respecting that env var here.